tools/node_http.js

```diff
@@ -3,7 +3,7 @@ const port = process.argv[2] || "4544";
 console.log("port", port);
 http
   .Server((req, res) => {
-    res.writeHead(200, { Connection: "close" });
+    res.writeHead(200);
```

Comment:
you don't need to call this one

Reply:
Removing that sped up Node quite a bit, I guess it makes it do an extra syscall. Thanks.